opennms-base-assembly/src/main/filtered/etc/datacollection/opennms.appliance.xml

```diff
@@ -0,0 +1,26 @@
+<datacollection-group xmlns="http://xmlns.opennms.org/xsd/config/datacollection" name="OpenNMS Appliances">
+   <systemDef name="OpenNMS Appliances (common)">
+      <sysoidMask>.1.3.6.1.4.1.5813.42.5.</sysoidMask>
+      <collect>
+         <includeGroup>mib2-host-resources-system</includeGroup>
+         <includeGroup>mib2-host-resources-memory</includeGroup>
+         <includeGroup>mib2-host-resources-storage</includeGroup>
```

Comment:
Just some food for thought, it might make sense to de-duplicate mib2-host-resources-storage vs. net-snmp-disk. Maybe we pick the one we can control a bit better with the `disk /` directive for mount points we are interested in?

Reply:
Fair point. I'll remove the MIB2-HR groups.